tests/expectations/tests/issue-643-inner-struct.rs

```diff
@@ -117,6 +117,54 @@ fn bindgen_test_layout_rte_ring() {
         8usize,
         concat!("Alignment of ", stringify!(rte_ring))
     );
+    assert_eq!(
```

Comment:
Yeah, these changes are wrong. They were done with modern clang, but these expectations are for older clang, so I think you just need to remove this change.

Reply:
Oh that's embarrassing. Fixed now.